packages/enr/src/raw_enr.ts

```diff
@@ -45,6 +52,18 @@ export class RawEnr extends Map<ENRKey, ENRValue> {
     }
   }
 
+  get rs(): ShardInfo | undefined {
+    const rs = this.get("rs");
+    if (!rs) return undefined;
+    return decodeRelayShard(rs);
+  }
+
+  get rsv(): ShardInfo | undefined {
+    const rsv = this.get("rsv");
+    if (!rsv) return undefined;
+    return decodeRelayShard(rsv);
+  }
```

Comment:
As discussed, these two fields used different decoding methods.

`rs` is used when less than 64 shards are supported, we can start with this one and ship lack it, shortly after adding support for `rsv`.